test/libsolidity/syntaxTests/nameAndTypeResolution/419_interface_structs.sol

```diff
@@ -3,5 +3,5 @@ interface I {
     }
 }
 // ----
-// Warning: (18-34): Defining empty structs is deprecated.
```

Comment:
Is there another test in this directory that defines a non-empty struct inside an interface?

Reply:
there is no specific test designated to ensuring empty structs are disallowed. I could create a new one. The numbers are just incremental (no further meaning), right?

Reply:
https://github.com/ethereum/solidity/blob/develop/test/libsolidity/syntaxTests/empty_struct.sol so I found it. I also removed the other companion `empty_struct_050.sol` since it's not going to be relevant either.

Reply:
This test case is about disallowing structs in interfaces. Please add a random member to the struct.

This test case may change when we enable structs in interfaces, though.